contributed_definitions/nyaml/NXcoordinate_system_set.yaml

```diff
@@ -88,138 +88,259 @@ doc: |
+        is inspected by looking towards the sample surface from a position
+        that is located behind the detector. For further information consult
+        also the help info for the xaxis_direction field.
+      enumeration: [undefined, north, east, south, west, in, out]
+    z_alias(NX_CHAR):
+      doc: |
+        Name or alias assigned to the z-axis base vector,
+        e.g. shortest edge.
+    z_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing z-axis base vector of
+        the sample surface reference frame. We assume the configuration
+        is inspected by looking towards the sample surface from a position
+        that is located behind the detector. For further information consult
+        also the help info for the xaxis_direction field.
+      enumeration: [undefined, north, east, south, west, in, out]
+  detector_reference_frameID(NXcoordinate_system):
+    doc: |
+      Details about the detector reference frame for a specific detector.
+    \@detector:
+      type: NX_CHAR
+      doc: |
+        Reference to the specifically named :ref:`NXdetector` instance
+        for which these conventions in this :ref:`NXprocess` group apply
+        (e.g. /entry1/instrument/detector1).
+    type(NX_CHAR):
+      doc: |
+        Type of coordinate system/reference frame used for
+        identifying positions in detector space Xd, Yd, Zd,
+        according to DOI: 10.1016/j.matchar.2016.04.008.
+      enumeration: [undefined, cartesian]
+    handedness(NX_CHAR):
+      doc: |
+        Handedness of the coordinate system if it is a Cartesian.
+      enumeration: [right_handed, left_handed]
+    origin(NX_CHAR):
+      doc: |
+        Where is the origin of the detector space reference
+        frame located. This is the location of Xd = 0, Yd = 0, Zd = 0.
+        Assume regions-of-interest in this reference frame form a
+        rectangle or cuboid.
+        Edges are interpreted by inspecting the direction of their
+        outer unit normals (which point either parallel or antiparallel)
+        along respective base vector direction of the reference frame.
+      enumeration: [undefined, front_top_left, front_top_right, front_bottom_right, front_bottom_left, back_top_left, back_top_right, back_bottom_right, back_bottom_left]
+    x_alias(NX_CHAR):
+      doc: |
+        Name or alias assigned to the x-axis base vector,
+        e.g. longest edge as some landmark on the detector.
+    x_direction(NX_CHAR):
+      doc: |
+        Direction of the positively pointing x-axis base vector of
+        the detector space reference frame. We assume the configuration
+        is inspected by looking towards the sample surface from a
+        position that is located behind the detector.
+        Different tools assume that different strategies can be used
+        and are perceived as differently convenient to enter
+        details about coordinate system definitions. In this ELN users
```

Comment:
There is no ELN here?

Reply:
Correct, implemented, thank you very much for your on point comments